lucid/launcher.py

```diff
@@ -93,15 +93,17 @@ def run(self):
             cli = lucid.utils.get_happi_client()
             devices = cli.search(beamline=self.beamline) or []
 
-            for dev in devices:
-                try:
-                    stand = get_happi_entry_value(dev, row_group_key)
-                    system = get_happi_entry_value(dev, col_group_key)
-                    dev_obj = happi.loader.from_container(dev, threaded=True)
-                    dev_groups[f"{stand}|{system}"].append(dev_obj)
-                except ValueError as ex:
-                    print(ex)
-                    continue
+            with lucid.utils.no_device_lazy_load():
+                for dev in devices:
+                    try:
+                        stand = get_happi_entry_value(dev, row_group_key)
+                        system = get_happi_entry_value(dev, col_group_key)
+                        dev_obj = happi.loader.from_container(dev,
+                                                              threaded=True)
+                        dev_groups[f"{stand}|{system}"].append(dev_obj)
+                    except ValueError as ex:
+                        print(ex)
```

Comment:
Let's take this as an opportunity to log it (also need module-level logger)

```suggestion
                        logger.exception('Failed to load device %s', dev)
```

Reply:
Addressed with https://github.com/pcdshub/lucid/pull/25/commits/45c3f0a0af85d901df1c2ae973fac7b59194aee5